pkg/graveler/committed/manager.go

```diff
@@ -14,13 +14,20 @@ import (
 type committedManager struct {
 	metaRangeManagers map[graveler.StorageID]MetaRangeManager
 	RangeManagers     map[graveler.StorageID]RangeManager
+	conflictResolvers []graveler.ConflictResolver
 	params            *Params
 }
 
-func NewCommittedManager(m map[graveler.StorageID]MetaRangeManager, r map[graveler.StorageID]RangeManager, p Params) graveler.CommittedManager {
+func NewCommittedManager(m map[graveler.StorageID]MetaRangeManager, r map[graveler.StorageID]RangeManager, crs []graveler.ConflictResolver, p Params) graveler.CommittedManager {
```

Comment:
Suggest that the catalog/graveler/committed will be created and work as the default behavior when created and optionally configured using a method to change the behavior.